lib/little-loader.js

```diff
@@ -51,27 +55,30 @@
    * @param {Object}    context   (Optional) Callback context
    * @returns {void}
    */
-  window._lload = function (src, callback, context) {
+  var _lload = function (src, callback, context) {
     /*eslint max-statements: [2, 20]*/
     context = context || this;
```

Comment:
I don't have a strong opinion. We should possibly consider [the strict/nonstrict `this` handling explained here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode#Securing_JavaScript). My basic feeling is if you pass `null|undefined|0|false|""` and expect that to do something useful, you're doing something weird and I don't mind clobbering it.